### DISCUSSION TITLE
Submit button does not work direct under editor (but does work above editor)

### DISCUSSION THREAD

**Discussion by ArvidDeJong** [livewire/flux#761] [Feature Requests]
### Issue with `<flux:editor />` and Submit Button Placement

When I place the following code directly **below** the `<flux:editor />`, the submit button does not work. However, if I place it **directly above** the editor, it works as expected.

```html
<flux:button type="submit" variant="primary">Save</flux:button>

**Comment by jeffchown** [comment]
@ArvidDeJong I just tried:

```
<form>
    <flux:editor />
    <flux:button type="submit" variant="primary">Save</flux:button>
</form>
```

And it submitted properly for me (below and above).

**Comment by alexcutts** [comment]
This was fixed in the latest version, updating should resolve it.

**Comment by ArvidDeJong** [comment]
https://github.com/user-attachments/assets/2ce3ea99-97b8-4bb4-b6c6-c827a490424c

Just updated to V1.0.28 and has no effect :(

**Comment by ArvidDeJong** [comment]
My mistake, after the update I had not cleared the cache yet.

I also renewed all elements with:
`php artisan flux:publish --all --force`

**Comment by joshhanley** [comment]
@ArvidDeJong this sounds more like an issue, not a feature request. So going to close this. Please open an issue if you are still experiencing the problem.
